shared/.pronto_spell.yml

```
@@ -1,4 +1,4 @@
-files_to_lint: '\.(cjsx|js|jsx|ts|tsx|haml|md|rb)$'
+files_to_lint: '\.(js|jsx|ts|tsx|haml|md|rb|erb|liquid)$'
```

Please make these alphabetically sorted again.
```suggestion
files_to_lint: '\.(erb|haml|js|jsx|liquid|md|rb|ts|tsx)$'
```

Also, I expect a spike of warnings in the future for Liquid tags like `endcomment`, those would need to be whitelisted (later).